about/press_kit/index.markdown

```diff
@@ -8,20 +8,7 @@ title: Press Kit
 ---
 # MoveIt Press Kit
 
- Welcome to the non-legally-binding introductory paragraph to the MoveIt Press Kit Terms of Use! We recognize that the following seem pedantic, but it’s important we have it because there have been legal issues before, and it’s important to us that we protect all the work everyone has done on MoveIt. We’ve done our best to make this readable, but it’s still legal-speak. Here's a  [Google open source case book](https://google.github.io/opencasebook/trademarks/) paragraph to help understand why there is a trademark on MoveIt, and what it does.
-
-
- `Trademarks serve a different purpose from copyrights and patents and can be licensed differently. Unlike copyright and patent laws, which were designed to incentivize creative works and useful inventions, trademark law’s purpose is to protect consumers by guaranteeing the source of a product or service.3 Copyrights and patent rights are property rights, whereas trademarks are fundamentally concerned with consumer protection. As such, an open source author may choose to openly license her project’s copyrights and patents while still enforcing her trademarks. Trademark enforcement is not inconsistent with open source principles: it restricts how a brand identifier can be used, rather than restricting what you can do with the open source code.4 Even Richard Stallman, the founder and president of the Free Software Foundation, recognizes that trademark enforcement is reasonable as long as software freedom is not restricted.5`
-
-
-### For Developers:
-
-The MoveIt trademark is the property of PickNik robotics, which can determine how the trademark is used as the owner. If this wasn't the case, another company could file a trademark, and then there would be legal issues with everyone's work. PickNik pursued this because ROS did it, and it seemed like a good idea to protect the name. While PickNik owns the trademark, they do not intend to enforce it for projects that operate within the spirit of open source, as listed below.
```

Comment:
> While PickNik owns the trademark, they do not intend to enforce it for projects that operate within the spirit of open source, as listed below.

I rather liked this sentence in the previous description. You effective took back the statement in your cleanup. :confused: